src/es/components/molecules/form/Form.js

```diff
@@ -65,17 +67,32 @@ export default class Form extends Shadow() {
         })
       }
     }
+
+    this.textAreaKeyUpListener = event => {
+      this.updateCounter(this.root.querySelector(':focus'))
+    }
   }
 
   connectedCallback () {
     if (this.shouldComponentRenderCSS()) this.renderCSS()
+    if (this.shouldComponentRenderHTML()) this.renderHTML()
     if (this.submit) this.submit.addEventListener('click', this.clickListener)
     if (this.hasAttribute('use-recaptcha') !== null) this.addEventListener('submit', this.submitListener)
+    this.textarea.forEach(a => {
```

Comment:
you added the event listener multiple times, for each textarea. Avoid this and only add it once on document, expl: https://github.com/mits-gossau/web-components-toolbox/blob/master/src/es/components/atoms/input/Input.js#L65

document.addEventListener('keyup', this.keydownListener)

just search if there is one element with maxlength and without no-counter, to figure out if you got to add the listener. Something like:

this.root.querySelector('[maxlength]:not([no-counter])')

Reply:
Mostly only one textarea